source/install/cloud-changelog.md

```diff
@@ -3,12 +3,62 @@
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
 Latest Mattermost Cloud releases:
+- [Release 2022-01-19](#release-2022-01-19)
 - [Release 2022-01-11](#release-2022-01-11)
 - [Release 2021-12-08](#release-2021-12-08)
 - [Release 2021-12-01](#release-2021-12-01)
 - [Release 2021-11-23](#release-2021-11-23)
 - [Release 2021-11-11](#release-2021-11-11)
-- [Release 2021-11-10](#release-2021-11-10)
+
+## Release 2022-01-19
+
+### Improvements
+
+#### User Interface (UI)
+ - The Renew Now button is no longer shown if the license id does not exist in the portal.
+ - Updated the About Mattermost Team Edition modal to change the community link from `mattermost.org` to `mattermost.com/community/`.
+ - Invite to team modal now auto-focuses its email search input.
+ - Added a new dialog for Remove License confirmation.
+ - Updated "Terms of Service" terminology to "Terms of Use" product-wide.
+ - Updated **Account Settings** terminology to **Settings**.
+ - Added Accept-Language header to generate link previews in the Default Server language.
+ - Added a new modal for the upload license workflow in the **System Console > Edition and License** page.
+ - The "Invite Members" button is now hidden when the Direct Message category is collapsed.
+
+#### Administration
+ - Plugins are re-initialized only when Plugin related config changes happen. This improves performance by avoiding plugin initialization which is quite costly.
+ - Added server support for receiving binary (messagepack encoded) websocket messages.
+ - Removed dead struct ``ManifestExecutables``.
+ - Added new flag ``showTeamSidebar`` in ``registerProducts``, which, when set to ``true``, displays the team sidebar in the product.
+ - Increased key length in plugin KV store to 150.
+ - Admins are now able to upgrade the server to the enterprise version and request the trial license with a single button click enhancing product adoption by an improved user experience.
+ - Added support for exporting and importing the type and ``edit_at`` of a post.
```

Comment:
```suggestion
 - Added support for exporting and importing the post type and ``edit_at`` post details.
```

Reply:
It isn't clear what "edit_at" post data refers to specifically. Is it possible to elaborate a bit further on this one to communicate the value of the change to users?